App/src/main/java/com/google/sps/servlets/User.java

```diff
@@ -0,0 +1,144 @@
+package com.google.sps.servlets;
+
+public class User {
```

Comment:
Oh whoah, how'd this get here? Did you merge in Cameron's changes? (A 4-day weekend means I might as well have been living under a rock in terms of how the codebase is coming lol.)